app/controllers/miq_ae_class_controller.rb

```diff
@@ -1660,6 +1661,20 @@ def x_show
     tree_select
   end
 
+  def refresh_git_domain
+    if params[:button] == "save"
+      git_based_domain_import_service.import(params[:git_repo_id], params[:git_branch_or_tag])
```

Comment:
Assuming a git repository is private to a tenant/user, some other tenant can fetch from it here, right?

Perhaps we don't care and consider git repository some kind of global objects, but I am curious whether, we don't care consciously :)